piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        void startGame(StartCallback callback);
+
+        RoundViewModel onNotesPlayed(CompletionCallback callback, Note... note);
+
+        interface StartCallback {
+            void onGameStarted(RoundViewModel viewModel);
+        }
+
+        interface CompletionCallback {
+            void onGameComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+    }
+
+    interface Presenter {
+
+        void onCreate();
+
+        void onNotesPlayed(Note... notes);
```

Comment:
In my mind, it'll be reducing the scope of the View's knowledge, keeping it dumber.

It's just naming anyway, since here the Presenter is an interface and it'd contain all the methods that I'm suggesting should be in the View, it won't make a big difference; just conceptually.